### DISCUSSION TITLE
close connection ?

### DISCUSSION THREAD

**Discussion by meaningg** [appy-one/acebase-client#56] [Q&A] [answered]
I open a connection to the database to make changes and then I want to close it, but I did not find something like db.close() in the documentation. I also see in the acebase-server logs that every time I call a function with a connection to the database, a new additional connection is created

**Comment by appy-one** [comment, marked as answer]
@meaningg if you are creating a new `AceBaseClient` instance each time you are getting/updating data, you will get multiple connections if you don't close them. Best practice is to create 1 `AceBaseClient` instance (a single connection to the server), reuse it in your app and keep it open. If you do want to close the connection however, use `client.disconnect()`

**Reply by meaningg** [reply]
Thank you so much!
